interface/src/ui/Stats.h

```diff
@@ -24,6 +24,8 @@ private: \
     type _##name{ initialValue };
 
 /**jsdoc
+ * The <code>Stats</code> API provides and statistics on Interface and domain operation, per the statistics overlay.
```

Comment:
```suggestion
 * The <code>Stats</code> API provides statistics on Interface and domain operation, per the statistics overlay. 
```

Please add additional note that explains the following: This API was written so that Hifi developers could more easily identify and track issues in Interface and the engine. It has been exposed in our API because it might be helpful to outside developers -- however, support and maintenance of this API will not be prioritized as its primary purpose is as an internal diagnosis tool.